Makefile

```
@@ -10,7 +10,10 @@ endif
 ifneq ($(origin DEFAULT_CHANNEL), undefined)
 BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
 endif
-BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL)
+ifneq ($(origin bundle_dir), undefined)
+BUNDLE_DIR = --output-dir=$(bundle_dir)
+endif
+BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL) $(BUNDLE_DIR)
```

Please use CAPITAL_LETTERS for all vars names, also touching the BUNDLE_METADATA_OPTS as it is a generated code and as such might change between operator-sdk's versions

DIverging from the original generated code might introduce complexity when we will have to follow the operator-sdk upgrade path

```suggestion
BUNDLE_METADATA_OPTS ?= $(BUNDLE_CHANNELS) $(BUNDLE_DEFAULT_CHANNEL) 

ifneq ($(origin OUTPUT_DIR), undefined)
BUNDLE_OUTPUT_DIR = --output-dir=$(OUTPUT_DIR)
endif
BUNDLE_FLAGS ?= $(BUNDLE_OUTPUT_DIR)
```

In the above suggestion, I have introduced BUNDLE_FLAGS which you will have to append to the end of the proper operator sdk command immediately after BUNDLE_METADATA_OPTS